<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Avogadro::QtPlugins::SupercellDialog</class>
 <widget class="QDialog" name="Avogadro::QtPlugins::SupercellDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Supercell Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Super Cell Options</string>
       </property>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="aRepeatLabel">
          <property name="text">
           <string>A repeat:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="aCellSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="bRepeatLabel">
          <property name="text">
           <string>B repeat:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="bCellSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="cRepeatLabel">
          <property name="text">
           <string>C repeat:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="cCellSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ok_cancel_bb">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok_cancel_bb</sender>
   <signal>accepted()</signal>
   <receiver>Avogadro::QtPlugins::SupercellDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok_cancel_bb</sender>
   <signal>rejected()</signal>
   <receiver>Avogadro::QtPlugins::SupercellDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
